components/brave_shields/browser/tracking_protection_service.cc

```diff
@@ -165,13 +181,11 @@ void TrackingProtectionService::ParseStorageTrackersData() {
     return;
   }
 
-  std::stringstream st(std::string(storage_trackers_buffer_.begin(), 
-    storage_trackers_buffer_.end()));
-  std::string tracker;
-
-  while(std::getline(st, tracker, ',')) {
-    first_party_storage_trackers_.push_back(tracker);
-  }
+  std::string trackers(reinterpret_cast<char*>(&storage_trackers_buffer_[0]),
```

Comment:
why do we need `reinterpret_cast<char*>`?
can't we just `std::string trackers(storage_trackers_buffer_.begin(), storage_trackers_buffer_.end())`

Reply:
Much better, updating. Thanks!